### DISCUSSION TITLE
What subscription services do you use?

### DISCUSSION THREAD

**Discussion by ghost** [NAIST-SE/AttractingContributors#422] [General]
Please add the subscription service you are using or would like to use.
I will add to your list too.

https://github.com/ozaki-shota/AttractingContributors_subscription_service

**Comment by aru1702** [comment]
I don't have particular subscription use right now, but if I have choose, I think I will pay for Youtube music subs so I can stream Youtube video as music using my phone.
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by ghost** [comment]
I think YouTube music subs are a good choice. I use Amazon's subscription service called Amazon Prime Student. It's not bad because it gives us access to many services.
